packages/cli/src/runners/task-managers/data-request-response-builder.ts

```diff
@@ -59,4 +64,23 @@ export class DataRequestResponseBuilder {
 			staticData: workflow.staticData,
 		};
 	}
+
+	private buildRunExecutionData(runExecutionData: IRunExecutionData) {
+		return {
+			startData: runExecutionData.startData,
+			resultData: runExecutionData.resultData,
+			executionData: runExecutionData.executionData
+				? {
+						contextData: runExecutionData.executionData.contextData,
+						metadata: runExecutionData.executionData.metadata,
+
+						// These are related to workflow execution and are not something
+						// that are accessible by nodes, so we always omit them
+						nodeExecutionStack: [],
+						waitingExecution: {},
+						waitingExecutionSource: null,
```

Comment:
Why I added these to `DataRequestResponseBuilder` and not into `DataRequestResponseStripper`? At first I thought stripper would be better place, but since it does stripping only based on request parameters, I decided to move them here. Also I don't think the data request response needs to be so closely tied to the internal structures of WF execution.